### ISSUE TITLE
[Feature request] Add option to "Rout trafic 1st over Tor, then over VPN"

### ISSUE THREAD

**Issue by Nokia808** [jkotra/eOVPN#5]
This is the last very wanted feature by most VPN users, & currently only one companies support it by it's own application:
https://airvpn.org/

It will be very great to add such feature in a generic program like yours, so that it could be applied for every VPN company !!

Regarding flatpak version in specific, it will be great if can make it utilize the system Tor bundle that should be already installed by user from package manager on her/his PC. However, I'm not sure if this is possible or not. I'm only sure 100% that your can rout Internet traffic 1st over system Tor bundle before OpenVPN because I'm already installed i2pd GUI on my PC as flatpak from FlatHub & the outproxy option for SOCKS proxy connection is already set by default to be Tor system bundle (address: 127.0.0.1 / Port: 9050). But is it possible to re-rout the output from Tor system bundle to your flatpak in easy way ??? If not, then Tor bundle should be included within your flatpak package, but this will bear additional duty to update flatpak package every time new Tor bundle version released .(& failure to do this is bad thing) ......

Other point: does this feature if added, will interfere with planned "VPN Kill Switch" feature ??

**Comment by jkotra** [comment]
This is beyond application use scope. 

if this type of setup is required, you can always start tor on your system and then open `eOVPN` and connect to your desired server.

However, the process might get easier when(planned) eOVPN gets feature to run particular set of commands on connection and disconnection events. no ETA on this so don't hold your breath.

**Comment by Nokia808** [comment]
@jkotra 
Hi. Just to be sure that you did not misunderstand me: I did not talk about Tor/over/VPN scenario (which mean user traffic pass through VPN server 1st then leave it to enter through Tor network). No ! I'm talked about VPN/over/Tor scenario (which mean user traffic enter 1st through Tor network then exit from it to enter then after through VPN service).

The VPN/over/Tor set up is very difficult & in fact need to edit system Tor bundle in a way make it useless for other usage, & because this I suggest this feature. I did not talked it frankly, but in fact what I want is to include already custom Tor bundle within your flatpak package so that it will be possible to utilize this feature that increase user anonymity by hide it's real IP from VPN company & at the same way continue to use Tor system bundle for other useful purpose in it's usual way ..........

By the way, we have similar situation that treated by this way - see the following:
https://github.com/flathub/cc.retroshare.retroshare-gui/pull/6
Kindly, read the discussion within above link.

Kindly, re-think about this issue & take your time. 

By the way, I'm not need to this setting for my use, but I'm asking it because I read in Internet frequently requests from many peoples for it ...... So, I'm asked to add this feature for benefit of community ......
